app/assets/javascripts/discourse/templates/application.hbs

```diff
@@ -6,6 +6,7 @@
       {{custom-html "top"}}
     {{/if}}
     {{global-notice}}
```

Comment:
Can't this be simply added into global-notice, uneasy adding another component for everyone, its more work for ember to do on every page.